content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,155 @@
+
+The default **Caller Hierarchy** view shows the callers (or predecessors) of the target method and the frequency with which they appear in the call stack.
+
+To view the callees (or successors), click **Callee Hierarchy**.
+
+{{< img src="/developers/ide_integrations/idea/callee-hierarchy.png" alt="The callee hierarchy view" style="width:100%;" >}}
+
+Right-click on a method in the call tree to see options to navigate to the source editor or flame graph.
+
+## Flame graph
+
+A flame graph is a visualization of profiling samples that shows stack traces and their relative frequency during the sample period. The Datadog plugin aggregates data collected over the requested time frame and multiple individual profiles are aggregated together. Each individual profile covers a 60 second interval within the requested time frame.
+
+{{< img src="/developers/ide_integrations/idea/flamegraph.png" alt="A flame graph showing CPU Time over the last 4 hours" style="width:100%;" >}}
+
+Each time you change the profile type, the time frame, or the environment, the Datadog plugin generates a new flame graph.
+
+You can navigate the flame graph in several ways:
+- Double-click on any frame to focus on that method and all the methods that it has called during the sampling period.
+- Use the minimap to pan around the graph.
+- Right-click on a method and select **Jump to Source** to go to the corresponding point in the source code.
+
+Hovering over a method displays a tooltip with the following information:
+- The class name and method signature
+- The package name
+- The profiling metric value and percentage breakdown.
+
+Profiling samples include stack trace and line number information. Use the **Separate Flame Graph by** button to switch between separating frames by method or line number.
+
+{{< img src="/developers/ide_integrations/idea/separate-flamegraph-by.png" alt="Use the tooltip button to separate frames by method or line number" style="width:40%;" >}}
+
+## Datadog Insights
+
+When a Profiling tab is active, Datadog Insights adds code highlights to the source code editor margin. The Datadog plugin displays an icon in the editor margin and highlights code based on the active Profiling data.
+- Hover over the icon to see more information.
+- Click the icon to open the top list Profiling tab or open Profiling in Datadog.
+  {{< img src="/developers/ide_integrations/idea/interest-options.png" alt="Click the Datadog icon to open the Profiling data in a tab or in Datadog" style="width:100%;" >}}
+
+The active Profiling tab also affects the IDEA project tree view, which is annotated with the selected profile's metrics:
+{{< img src="/developers/ide_integrations/idea/project-tree-view.png" alt="The project tree annotated with profile metrics from a profile tab" style="width:60%;" >}}
+
+## Feedback
+
+Let us know what you think about the plugin! Provide feedback on our [discussion forum][1], or send an e-mail to `team-ide-integration@datadoghq.com`.
```

Comment:
maybe also open and reference a public slack channel?

Reply:
If someone creates a channel, I can add it the doc.